documentation/service_admin/proc-installing-using-ansible.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// assembly-installing-openshift.adoc
+
+include::common/attributes.adoc[]
+
+[id='installing-using-ansible_{context}']
+= Installing {ProductName} using Ansible
+
+Installing {ProductName} using Ansible requires creating an inventory file with the variables for configuring the
+system. Example inventory files can be found in the `ansible/inventory` folder. For more information about the supported Ansible configuration settings see xref:ref_ansible-config-settings_{enmasse}[Ansible configuration settings].
```

Comment:
Remove the { } around enmasse 

xref:ref_ansible-config-settings_enmasse[Ansible configuration settings].

Reply:
Fixed